src/sdk/SpeechTranslationConfig.ts

```diff
@@ -92,26 +91,6 @@ export abstract class SpeechTranslationConfig extends SpeechConfig {
         return ret;
     }
 
-    /**
-     * Gets/Sets the authorization token.
-     * Note: The caller needs to ensure that the authorization token is valid. Before the authorization token
-     * expires, the caller needs to refresh it by calling this setter with a new valid token.
-     * @member SpeechTranslationConfig.prototype.authorizationToken
-     * @function
-     * @public
-     * @param {string} value - The authorization token.
-     */
-    public abstract set authorizationToken(value: string);
-
-    /**
-     * Gets/Sets the speech recognition language.
-     * @member SpeechTranslationConfig.prototype.speechRecognitionLanguage
-     * @function
-     * @public
-     * @param {string} value - The authorization token.
-     */
-    public abstract set speechRecognitionLanguage(value: string);
```

Comment:
We can't remove methods from the SDK API surface without a major version revision, and we're not ready to move the SDK to 2.0 yet, so these should stay.

Reply:
SpeechTranslationConfig extends SpeechConfig which already declares/defines these properties and methods.